tests/test_anvl.py

```diff
@@ -29,15 +29,14 @@ def test_strips_left_whitespace_of_value(self):
         expected = {'key': 'value '}
         assert actual == expected
 
-    @pytest.mark.xfail
     def test_string_without_colon(self):
         """
         IndexError is not an appropriate response and
```

Comment:
This description is no longer exactly accurate, as we're raising a different type of error than the one specified.